Sources/FoundationEssentials/String/BuiltInUnicodeScalarSet.swift

```diff
@@ -52,7 +52,7 @@ internal struct BuiltInUnicodeScalarSet {
 
     // CFUniCharIsMemberOf
     func contains(_ scalar: Unicode.Scalar) -> Bool {
-        let planeNo = Int((scalar.value >> 16) & 0xFF)
+        let planeNo = Int((scalar.value >> 16) & UInt32(0xFF))
```

Comment:
Does `Unicode.Scalar.value` have a different type on Android than on other platforms?

Reply:
No, it's just that the new operator conflicts during overload resolution, with this specific instance of using `&` on a constant that is not typed.

Reply:
Hmm, based on that it seems like the new operator has the possibility of a widespread affect across the project, and I worry that changes like this which appear arbitrary to someone not knowledgeable of the android build are likely to cause the android build to be broken quite frequently. Is there anything we can do to target that operator to just the uses we want? How unruly would it be to perform that at each bitwise-and with a `mode_t` rather than using a global operator?

Reply:
that should work too, let me see how invasive such change would be.